From 1a33c2baf58bc0d1933254bc80b01f33fef15630 Mon Sep 17 00:00:00 2001
From: "Bryn M. Reeves" <bmr@redhat.com>
Date: Mon, 19 May 2014 20:51:12 +0100
Subject: [PATCH 69/72] Add oVirt Data Warehouse support

Add support for DWH to the oVirt plugin. Based on a patch for
sos-2.0 by Sandro Bonazzola. Fixes Issue #236.

Signed-off-by: Bryn M. Reeves <bmr@redhat.com>
---
 sos/plugins/ovirt.py | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/sos/plugins/ovirt.py b/sos/plugins/ovirt.py
index c6f8957..f4b7f1e 100644
--- a/sos/plugins/ovirt.py
+++ b/sos/plugins/ovirt.py
@@ -35,8 +35,8 @@ class Ovirt(Plugin, RedHatPlugin):
         pattern=r"""
         ^
         /etc/
-        (rhevm|ovirt-engine)/
-        engine.conf
+        (rhevm|ovirt-engine|ovirt-engine-dwh)/
+        (engine.conf|ovirt-engine-dwhd.conf)
         (\.d/.+.conf)?
         $
         """
@@ -44,7 +44,7 @@ class Ovirt(Plugin, RedHatPlugin):
 
     DEFAULT_SENSITIVE_KEYS = (
         'ENGINE_DB_PASSWORD:ENGINE_PKI_TRUST_STORE_PASSWORD:'
-        'ENGINE_PKI_ENGINE_STORE_PASSWORD'
+        'ENGINE_PKI_ENGINE_STORE_PASSWORD:DWH_DB_PASSWORD'
     )
 
     plugin_name = "ovirt"
@@ -76,7 +76,9 @@ class Ovirt(Plugin, RedHatPlugin):
         self.add_copy_specs([
             "/etc/ovirt-engine",
             "/etc/rhevm",
+            "/etc/ovirt-engine-dwh",
             "/var/log/ovirt-engine",
+            "/var/log/ovirt-engine-dwh",
             "/var/log/rhevm",
             "/etc/sysconfig/ovirt-engine",
             "/usr/share/ovirt-engine/conf",
-- 
1.9.3

